src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -39,30 +37,32 @@
                 new Address("1 Jurong East Street 32, #08-111"),
                 new PostalCode("609477"),
                 new Deadline(LocalDate.parse("31-12-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                    getTagSet("4 bedrooms", "No need for renovation")),
+                new Client(new Name("Bob"), new Contact("98664535"),
+                    new Email("bob@gmail.com"), new AskingPrice("$800,000")),
+                getTagSet("4 bedrooms", "No renovation")),
             new Property(new Name("Burghley Drive"),
                 new Type("Landed"),
                 new Address("12 Burghley Drive"),
-                new PostalCode("123456"),
-                new Deadline(LocalDate.parse("21-07-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                new Remark("99 year leasehold, lowest selling price is $5,040,0000"),
-                    getTagSet("99 year leasehold")),
+                new PostalCode("558977"),
+                new Deadline(LocalDate.parse("31-07-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Remark("Lowest selling price is $5,040,0000"),
+                getTagSet("99 year leasehold", "Balcony")),
             new Property(new Name("Woodlands Crescent"), new Type("Hdb"),
-                    new Address("Blk 784 Woodlands Crescent #01-01"),
-                    new PostalCode("730784"),
-                    new Deadline(LocalDate.parse("01-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                    new Client(new Name("Alice"), new Contact("91234567"),
-                            new Email("alice@gmail.com"), new AskingPrice("$800,000")),
-                    getTagSet("2 bedrooms", "65 square metres")),
+                new Address("Blk 784 Woodlands Crescent #01-01"),
+                new PostalCode("731784"),
+                new Deadline(LocalDate.parse("01-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Client(new Name("Caleb"), new Contact("84459627"),
+                    new Email("caleb_goh@gmail.com"), new AskingPrice("$350,000")),
+                getTagSet("2 bedrooms", "65 square metres")),
```

Comment:
do u think `ClientBuilder` should be used for this?

Reply:
I think we should probably separate source code and test code

Reply:
ok